app/views/staff/session_formats/create.js.erb

```diff
@@ -11,4 +11,5 @@
   $('#config-modal .errors').append('<%=  session_format.errors.full_messages.join(", ") %>');
```

Comment:
@gitcow Hmmm, actually I am not getting this behavior, locally or in the review app. Can you tell me exactly what you did for this to happen?
Well, to clarify: I don't get this behavior for tracks or session formats. However errors do *not* clear on the rooms modal, but only when you exit by clicking 'cancel'. I remember hearing that that whole section is going to be reworked though, so I'm not sure how much time I should spend on that one issue? I have tried a bunch of things to fix it but so far nothing has worked.